README.md

```diff
@@ -1 +1,177 @@
-# DeepStack-CameraUI
\ No newline at end of file
+# DeepStack-CameraUI
+DeepStack-CameraUI is a real-time computer vision web-application. Powered by Deepstack and Streamlit, this application performs realtime AI processing such as **object detection**, **face recognition**, **face detection** and **custom detection** on video frames from WebCam, Wired Camera or IP camera through an interactive web interface.
+
+## DeepStack-CameraUI Interface
+The interface of this web application is composed of two sections the **side-bar** which is the place where all configurations are set up and the **main interface** where the live detections are displayed.
+
+### Deepstack-CameraUI Settings
+The side-bar compresses different elements detailed below:
+
+* **Choose the APIs**
+
+This widject allows you to select which API techniques delivered by this application to run among the following **object detection**, **face recognition**, **face registration**, **face deregistration**, **face detection** and **custom detection**.
+
+* **DeepStack URL**
+
+To connect to Deepstack server the user must provide the right URL to Deepstack like `http://localhost:80` or `http://localhost:5670` depending on the URL port. 
+Fro more information check the documentation https://docs.deepstack.cc/
+
+* **Camera Address**
+
+This is the crucial part of this application, the user must supply all the information related to his IP canera such **User name**, **password**, **IP address**, **port** and **media**.
```

Comment:
Camera address can be a `WebCam` or `IP Camera`. You need to mention both and give example

- WebCam: Specify 0,1 or the index of the camera as the camera address if you are using a WebCam or any other cable connected to your local machine (for  Ubuntu, the camera index start from `-1` )
- IP Camera: IP camera URL can require credentials or not.  A sample is which doesn't have/require other credentials  is `http://180.234.23.1:9090/video` and a sample which requires credentials is `http://username:password@180.234.23.1:9090/video` 
-

Then you can break down the parts of the IP camera. Also mention that they should refer to the IP cametra instuctions on how to access the video and paste the video url in the browser to confirm it loading the live feed before feeding into DeepStack-CameraUI